4/Dockerfile

```diff
@@ -3,13 +3,20 @@ FROM centos/s2i-base-centos7
 # This image provides a Node.JS environment you can use to run your Node.JS
 # applications.
 
-EXPOSE 8080
-
 # Add $HOME/node_modules/.bin to the $PATH, allowing user to make npm scripts
 # available on the CLI without using npm's --global installation mode
 # This image will be initialized with "npm run $NPM_RUN"
 # See https://docs.npmjs.com/misc/scripts, and your repo's package.json
 # file for possible values of NPM_RUN
+# Description
+# Environment:
+# * $HOME_APP_ROOT - Root directory of application
+# * $NPM_RUN - Select an alternate / custom runtime mode, defined in your package.json files' scripts section (default: npm run "start").
+# Expose ports:
+# * 8080 - Unprivileged port used by nodejs application
+# Additional packages
+# * findutils are needed to help fix permissions
```

Comment:
findutils are getting installed in s2i-core so this comment does not make sense in here.